doc/use_cli_effectively.md

```diff
@@ -122,7 +122,7 @@ az rest --method PATCH
         --body '{"web":{"redirectUris":["https://myapp.com"]}}'
 ```
 
-## Quoting issues
+## Quoting issues - object/array parameter value
```

Comment:
In my mind, this phrase `object/array parameter value` is unclear to the user what it means. What is an "object parameter"? What is an "array parameter"? It depends on specific commands. 

Also, this section is about quoting, instead of `passing object/array parameter value` (if I understand the meaning correctly.)

The change should go to doc https://docs.microsoft.com/en-us/cli/azure/group/deployment?view=azure-cli-latest which is where customer raised https://github.com/Azure/azure-cli/issues/10731, instead of this section.

Reply:
As discussed, let's add a separate section dedicated to describe list/JSON object passing and link to this section with:

> Some quoting rules apply in different shells, please see [Quoting issues](#quoting-issues).

Reply:
Actually I am think this may be related to https://github.com/Azure/azure-cli/issues/13170. We can make `--tags` support JSON as well to make it much easier for customers.